combo_cases_and_deaths/delphi_combo_cases_and_deaths/run.py

```diff
@@ -107,25 +141,56 @@ def combine_usafacts_and_jhu(signal, geo, date_range, issue_range=None, fetcher=
         geo_to_fetch,
         issues=issue_range
     )
+
     if check_none_data_frame(usafacts_df, "USA-FACTS", date_range) and \
-       (geo_to_fetch not in ('state', 'county') or
+       (geo_to_fetch not in ('state', 'county', 'msa') or
         check_none_data_frame(jhu_df, "JHU", date_range)):
+        return None
+
+    merged_df = merge_dfs_by_geos(usafacts_df, jhu_df, geo_to_fetch)
+    unique_dates = merged_df["timestamp"].unique()
+    return unique_dates
+
+
+def combine_usafacts_and_jhu(signal, geo, date_range, issue_range=None, fetcher=covidcast.signal):
+    """Add rows for PR from JHU signals to USA-FACTS signals.
+
+    For hhs and nation, fetch the county `num` data so we can compute the proportions correctly
+    and after combining JHU and USAFacts and mapping to the desired geos.
+    """
+    is_special_geo = geo in ["hhs", "nation"]
```

Comment:
this occurs at least twice; probably time to put it into a function

Reply:
ah, looks like i didn't actually need it the second time.